### DISCUSSION TITLE
What's Next.

### DISCUSSION THREAD

**Discussion by americanseeder1865** [UltimaHoarder/UltimaScraper#1598] [General]
Seems like Onlyfans is killing themselves. I do still think there is a place for scrapers because someone else is going to pop up a site or sites, as the content creators are making waaaaayyyy too much money. Attn: @DIGITALCRIMINAL and other developers.

**Comment by UltimaHoarder** [comment]
There's already a discussion on it and I replied to it

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by americanseeder1865** [comment]
woops.

https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1596
